### PR TITLE
update 1.5.7

### PR DESCRIPTION
**Pull request by Vadya963**
please update, I tried

### CODE REVIEW

File: src/defs/client.ts [hunk 1]
```diff
@@ -834,8 +825,8 @@ tmpDef.scriptSide = ScriptSide.Client;
 ClientDefinitions.push(tmpDef);
 
 tmpDef = new MTAFunction;
-tmpDef.label = "getLocalPlayer";
-tmpDef.description = "This function gets the player element of the client running the current script.";
+tmpDef.label = "localPlayer";
```
Comment (Subtixx) [inline src/defs/client.ts:828]: localPlayer is not an MTA function.

File: src/defs/client.ts [hunk 2]
```diff
@@ -923,15 +914,6 @@ tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Client;
 ClientDefinitions.push(tmpDef);
 
-tmpDef = new MTAFunction;
```
Comment (Subtixx) [inline src/defs/client.ts:926]: Why was this removed?
Wiki doesn't say it has been removed from MTA:
https://wiki.multitheftauto.com/wiki/SetElementMatrix

File: src/defs/client.ts [hunk 3]
```diff
@@ -1301,15 +1283,6 @@ tmpDef.argDescs = {};
...
-tmpDef.label = "setPlayerHudComponentVisible";
```
Comment (Subtixx) [inline src/defs/client.ts:1305]: Again why was this removed? MTA Wiki doesn't mention it has been:
https://wiki.multitheftauto.com/wiki/SetPlayerHudComponentVisible

**Review by Subtixx — APPROVED** [review]

File: src/defs/client.ts (hunk 2 above)
Comment (Vadya963) [reply src/defs/client.ts:926]: I moved it to shared

File: src/defs/client.ts (hunk 1 above)
Comment (Vadya963) [reply src/defs/client.ts:828]: this can be fixed

File: src/defs/client.ts (hunk 3 above)
Comment (Vadya963) [reply src/defs/client.ts:1305]: I moved it to shared.

Comment (Vadya963) [reply src/defs/client.ts:1305]: if the function has two syntaxes and you want to use the server syntax, the client syntax is displayed
